releasenotes/notes/fix-ci-app-api-call-12f848d25f93d02d.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    CI App: This fix resolves an issue where the tracer was doing extra requests if the ``DD_CIVISIBILITY_ITR_ENABLED`` env var was not set.
```

```suggestion
    CI Visibility: This fix resolves an issue where the tracer was doing extra requests if the ``DD_CIVISIBILITY_ITR_ENABLED`` env var was not set.
```
Just to keep consistency with the previous changes, I think we should continue calling this scope `CI Visibility` 😄